src/discussion-tool-selector/DiscussionToolSelectorContainer.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import DiscussionToolSelector from './discussion-tool-selector/DiscussionToolSelector';
+
+// XXX this is just for testing and should be removed ASAP
```

Comment:
when is ASAP? is there a follow-up ticket to clean this up?

Reply:
Almost immediately - this is basically test data and needs to be fed from the server. The next step is to hide this test data behind a client-side API layer here in the app, and once that infra is in place, actually load it from an API.

Reply:
OK. It'd be cool to see it explicitly referenced somewhere so we don't forget but I'll believe you that it's gonna happen

Reply:
Yeah, I mean, this is a key piece of functionality that's just mocked right now - I also didn't write this comment, fwiw. :)  This will be covered by our tasks to back this with API calls that tell us the available tools.

Reply:
Yeah, and fwiw, I had basically asked them to minimally mock this up like so;
I don't think we'll forget to wire up the API :)